src/storage-client/src/types/sources.proto

```diff
@@ -214,7 +214,7 @@ message ProtoPostgresSourcePublicationDetails {
 
 message ProtoLoadGeneratorSourceConnection {
     oneof generator {
-        google.protobuf.Empty counter = 1;
+        ProtoCounterLoadGenerator counter = 1;
```

Comment:
Is it safe to reuse ids?

Reply:
https://materializeinc.slack.com/archives/C0246GEHL8N/p1675971029060749?thread_ts=1675971007.308549&cid=C0246GEHL8N assuming this isn't "one of the few that is persisted to disk", though I have no idea.

Reply:
I also don't know. Can you figure this out, or do the normal reserved thing if that's too hard/slow/annoying?

Reply:
Dunno how to do that without auditing the entire codebase, so I went ahead and just reserved the old field number.